client/components/jetpack/card/jetpack-rna-action-card/index.tsx

```diff
@@ -89,6 +89,7 @@ const JetpackRnaActionCard: React.FC< RnaActionCardProps > = ( {
 						rel={ ctaButtonExternal ? 'noopener noreferrer' : '' }
 					>
 						{ ctaButtonLabel }
+						{ ctaButtonExternal && ' ' }
```

Comment:
Just making sure this was intentional.

Reply:
Yes it is. See the last point in the proposed changes:

> Fix spacing in the RNA CTA which had no space between text and external icon